src/DataSets/DataSets.component.js

```diff
@@ -303,21 +332,37 @@ const DataSets = React.createClass({
 
         const logActions = [
             <FlatButton
```

Comment:
A question about the UX, @adrianq : does it make sense to move this action inside the logs box, as center button? this way the user only sees it when it really makes sense.